contracts/ColonyNetworkStorage.sol

```diff
@@ -70,4 +70,7 @@ contract ColonyNetworkStorage is DSAuth {
   uint256 reputationRootHashNNodes;
   // Mapping containing how much has been staked by each user
   mapping (address => uint) stakedBalances;
+
+  // Mapping containing the last auction start timestamp for a token address
+  mapping (address => uint) recentAuctions;
```

Comment:
Given that the auction contracts selfdestruct when complete, we'd have to check that there was still a contract there before trying to call `startTime` on it, otherwise we'd revert. That's fairly straightforward to do in assembly with the `extcodesize` call though.

Reply:
ah yes, you're right. But we probably won't be able to determine if a month has gone by if the auction contract is destroyed, right?

Reply:
Ah, also true, I hadn't considered that an auction could finish within the 30 days.

Reply:
Let's leave it as is then. At most we might add the auction contract address record to this mapping at some point.